test/solr/search_record_test.rb

```diff
@@ -59,6 +59,8 @@ class SearchRecordTest < ActiveSupport::TestCase
       #with(:updated_month, month) if month.present?
       #paginate :page => 1, :per_page => 10
     end
+puts solr_search_1.results
```

Comment:
Hmm, i have to wrap up for the day, but my attempt to output the results here didn't turn into anything. We could try `puts solr_search_1.results.inspect` instead...?